ArduCopter/Copter.h

```diff
@@ -472,6 +472,7 @@ class Copter : public AP_HAL::HAL::Callbacks {
     // Current location of the copter (altitude is relative to home)
     Location_Class current_loc;
     Location ekf_origin;
+    Vector3f current_pos;
```

Comment:
Let's consider carefully if we want to maintain this current_pos variable which is just a copy of what's in the EKF.  Maybe we should get input from @peterbarker and @OXINARF.